src/game/Game.java

```diff
@@ -302,52 +302,48 @@ public void printResults() {
                         while (board[row + 1][col] == opColor && row < 6) {
                             row++;
                         }
-                        if (board[row + 1][col] == EMPTY) {
-                            move[0] = row + 1;
-                            move[1] = col;
-                            moves.add(move);
+                        if (board[row + 1][col] == EMPTY && board[row][col] == opColor) {
+                            moves.add(new int[]{row + 1, col});
```

Comment:
We need to create a new move array each time.
Otherwise we just keep rewriting the previous one, the one already added to move (reference to same object).